bool IsObstacle
float32 Distance
geometry_msgs/Pose ObstaclePose